proto/iotics/api/meta.proto

```diff
@@ -29,6 +29,12 @@ service MetaAPI {
   //   choose to end the stream early once they have received enough results. (E.g. in the case of Scope.LOCAL this
   //   would be after the one and only sequence of chunks has been received.)
   rpc SparqlQuery(SparqlQueryRequest) returns (stream SparqlQueryResponse) {}
+
+  // SparqlUpdate performs a SPARQL 1.1 query with write access to a local custom metadata graph and returns a result. 
+  // Note that:
+  // - After the first request chunk has been streamed to the server, subsequent chunks may omit headers as these will
+  //   be inferred from the first chunk and won't be interleaved with chunks from other requests.
```

Comment:
- I think the note about interleaving does not apply here (since it's exactly one request that's being "uploaded").